js/ui/color_box/color_box.js

```diff
@@ -302,7 +302,8 @@ const ColorBox = DropDownEditor.inherit({
 
     _renderValue: function() {
         const value = this.option('value');
-        this.option('text', this.option('editAlphaChannel') ? colorUtils.makeRgba(value) : value);
+
+        this.option('text', (this.option('editAlphaChannel') && value !== null) ? colorUtils.makeRgba(value) : value);
```

Comment:
What about `undefined`?

Reply:
Now the _undefined_ value sets as "rgba(0, 0, 0, 1)" in the same way as before my fix. The _undefined_ value is unexpected for the widget and i think it is no reasons to change behavior that works since 18.1.6

Reply:
I agree, thx